addons/chemical/functions/fnc_breathing.sqf

```diff
@@ -0,0 +1,47 @@
+#include "script_component.hpp"
+
+/*
+ * Author: DiGii
+ * 
+ * Arguments:
+ * 0: Target <OBJECT>
```

Comment:
shouldnt that be player or unit? target sounds like its called on other player from caller